src/main/java/in/projecteka/consentmanager/user/LockedUserService.java

```diff
@@ -0,0 +1,87 @@
+package in.projecteka.consentmanager.user;
+
+import in.projecteka.consentmanager.clients.ClientError;
+import in.projecteka.consentmanager.user.model.LockedUser;
+import lombok.AllArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Mono;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@AllArgsConstructor
+public class LockedUserService {
+    private final LockedUsersRepository lockedUsersRepository;
+    private final LockedServiceProperties lockedServiceProperties;
+    private final Logger logger = LoggerFactory.getLogger(LockedUserService.class);
+
+    public Mono<LockedUser> userFor(String patientId) {
+        logger.info("Invoking repository go get the locked user for patientId " + patientId);
```

Comment:
It need not be info log. It could be a debug log.

Reply:
done